jenkins/test-history/gen_html_test.py

```diff
@@ -27,83 +28,121 @@
 
 TEST_DATA = {
     'test_names': ['test1', 'test2'],
-    'buckets': {'gs://kubernetes-jenkins/logs/': {
-        'kubernetes-release': {
-            '3': {'tests': [{'name': 0, 'time': 3.52}]},
-            '4': {'tests': [{'name': 0, 'time': 63.21, 'failed': True}]}},
-        'kubernetes-debug': {
-            '5': {'tests': [{'name': 0, 'time': 7.56}]},
-            '6': {'tests': [
-                {'name': 0, 'time': 8.43},
-                {'name': 1, 'failed': True, 'time': 3.53}]}},
-}}}
-
-TEST_BUCKETS_DATA = {
-    'gs://kubernetes-jenkins/logs/': { 'prefix': '' },
-    'gs://bucket1/': { 'prefix': 'bucket1_prefix' },
-    'gs://bucket2/': { 'prefix': 'bucket2_prefix' }
+    'buckets': {
+        'gs://kubernetes-jenkins/logs/': {
+            'kubernetes-release': {
+                '3': {'tests': [{'name': 0, 'time': 3.52}]},
+                '4': {'tests': [{'name': 1, 'time': 63.21, 'failed': True}]},
+            },
+            'kubernetes-debug': {
+                '5': {'tests': [{'name': 0, 'time': 7.56}]},
+                '6': {'tests': [
+                    {'name': 0, 'time': 8.43},
+                    {'name': 1, 'failed': True, 'time': 3.53},
+                ]},
+            },
+        },
+        'gs://rktnetes-jenkins/logs/': {
+            'kubernetes-release': {
+                '7': {'tests': [{'name': 0, 'time': 0.0, 'skipped': True}]},
+            },
+        },
+        'gs://kube_azure_log/': {},
+    },
 }
 
+TEST_BUCKETS_DATA = """{
```

Comment:
I suppose it didn't really need to be, just need to do a `json.dumps` where the string is expected. Fixed.